src/app/components/radio/index.html

```diff
@@ -4,57 +4,103 @@
     </sky-demo-page-property>
-
-<!--    <sky-demo-page-property
-        propertyName="tabindex"
-        defaultValue="0"
+    <sky-demo-page-property
+      propertyName="tabIndex"
+      defaultValue="0"
+      isOptional="true"
     >
-      Specifies ...
+    This property is deprecated. It specifies an index for the radio button. If the index is not defined, it is set to the position of the radio button on load.
     </sky-demo-page-property>
-
     <sky-demo-page-property
-        propertyName="name"
+      propertyName="value"
     >
-      Specifies ...
+    Specifies a value for the radio button. The value usually corresponds to the radio button's label.
     </sky-demo-page-property>
+  </sky-demo-page-properties>
 
+  <sky-demo-page-properties sectionHeading="Radio button events">
     <sky-demo-page-property
-        propertyName="value"
+      propertyName="change"
     >
-      Specifies ...
+    Fires when users select a radio button.
     </sky-demo-page-property>
+  </sky-demo-page-properties>
 
+  <sky-demo-page-properties sectionHeading="Radio button group properties">
+    <p>
+      The radio button group component's <stache-code>sky-radio-group</stache-code> element wraps <stache-code>sky-radio</stache-code> elements to organize the radio buttons into a group. It is required for SKY UX radio buttons on Angular reactive forms to manage the selected value and keep the form up-to-date. We recommend using the radio button group component with all radio buttons.
+    </p>
     <sky-demo-page-property
-        propertyName="inputId"
+      propertyName="ariaLabelledBy"
+      isOptional="true"
     >
-      Specifies ...
+      Sets the radio button group's <stache-code>aria-labelledby</stache-code> attribute to support accessibility. The value should be the HTML element ID (without the leading #) of the element that labels the radio button group. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="name"
+      isOptional="true">
+      Specifies a name for the collection of radio buttons that the component groups together. If a <stache-code>name</stache-code> property is also set on the <stache-code>sky-radio</stache-code> elements, then this name overwrites it. If a <stache-code>name</stache-code> property is not set on the <stache-code>sky-radio</stache-code> elements, then this property is required.
```

Comment:
Sorry, I know we talked about this, but I've been thinking about it and do you think it might be better to list this as required (isOptional=false) and list the radio group and it's properties first on the page? So that this is the first thing they see? If not then this is fine :)

I've also been wondering about the ordering of properties in general and despite knowing what happened the last time I asked about the ordering of properties in documentation 😜 , I kind of have to know: Have we considered listing required properties first and then optional ones? It seems like it would be more straightforward to look and see what I must have on the component to use it.

Reply:
Sorry for the slow response on this. I agree with both of your suggestions, and I followed up with Steve who also agreed. So I removed the "optional" label from the name property, moved the group properties to the top of the page, and reorganized the properties so that required properties appear in alphabetical order followed by optional properties in alphabetical order. Going forward, I'll make that same change to properties in other component docs as well.

Thanks for the feedback. ... With those changes in place, does this look OK to merge?